src/app/public/modules/auth-http-client/auth-interceptor.spec.ts

```diff
@@ -1,87 +1,48 @@
 //#region imports
 
-import {
-  HttpEventType,
-  HttpHandler,
-  HttpParams,
-  HttpRequest,
-  HttpSentEvent
-} from '@angular/common/http';
-
-import {
-  Observable
-} from 'rxjs/Observable';
-
+import { HttpHandler, HttpParams, HttpRequest } from '@angular/common/http';
```

Comment:
Imports should be formatted as one type per line, and in alphabetical order. 
See: https://docs.blackbaud.com/skyux-team-docs/style#imports

Reply:
Fixed.